render/json.go

```diff
@@ -112,3 +118,29 @@ func (r JsonpJSON) Render(w http.ResponseWriter) (err error) {
 func (r JsonpJSON) WriteContentType(w http.ResponseWriter) {
 	writeContentType(w, jsonpContentType)
 }
+
+func (r AsciiJSON) Render(w http.ResponseWriter) (err error) {
+	r.WriteContentType(w)
+	ret, err := json.Marshal(r.Data)
+	if err != nil {
+		return err
+	}
+
+	result := ""
+	for _, r := range string(ret) {
+		cvt := ""
+		if r < 128 {
+			cvt = string(r)
+		} else {
+			cvt = fmt.Sprintf("\\u%04x", int64(r))
+		}
+		result = result + cvt
```

Comment:
conside `bytes.Buffer`? https://stackoverflow.com/questions/1760757/how-to-efficiently-concatenate-strings-in-go/23857998

Reply:
done